rapidfuzz/fuzz.hpp

```diff
@@ -127,8 +127,8 @@ struct CachedPartialRatio {
     CachedPartialRatio(InputIt1 first1, InputIt1 last1);
 
     template <typename Sentence1>
-    CachedPartialRatio(const Sentence1& s1)
-        : CachedPartialRatio(common::to_begin(s1), common::to_end(s1))
+    CachedPartialRatio(const Sentence1& sent)
```

Comment:
In other places I simply appended a underscore to prevent this warning. For consistency please use `s1_` here.

Reply:
Okay. I usually use the underscore suffix approach for private member variables as well. I wasn't sure about the change in your library because I saw some other classes not applying the underscore style. I will double check on those as well.

Reply:
did you mean to append an underscore to the constructor parameter `s1` or the class member variables?

Reply:
For others I simply appended it to the constructor parameter (see e.g. CachedHamming)

Reply:
done.